lib/radiator/auth/register.ex

```diff
@@ -39,6 +41,16 @@ defmodule Radiator.Auth.Register do
     end
   end
 
+  def find_users(query_string \\ "") do
```

Comment:
I think my intent is eventually to disallow this case altogether ^^. Managing users in general is probably a separate instance-admin-api. But since I forgot to sort them I think I should do another case here for now.